hoot-core/src/main/cpp/hoot/core/io/OgrUtilities.cpp

```diff
@@ -54,7 +54,8 @@ void OgrUtilities::loadDriverInfo()
   _drivers.push_back(OgrDriverInfo(".db",       "SQLite",         true,     true,   GDAL_OF_VECTOR));
   _drivers.push_back(OgrDriverInfo(".mif",      "MapInfo File",   true,     true,   GDAL_OF_VECTOR));
   _drivers.push_back(OgrDriverInfo(".tab",      "MapInfo File",   true,     true,   GDAL_OF_VECTOR));
-  _drivers.push_back(OgrDriverInfo(".s57",      "S57",            true,     true,   GDAL_OF_VECTOR));
+//  _drivers.push_back(OgrDriverInfo(".s57",      "S57",            true,     true,   GDAL_OF_VECTOR));
+  _drivers.push_back(OgrDriverInfo(".000",      "S57",            true,     true,   GDAL_OF_VECTOR));
```

Comment:
Is there a reason to turn off the `.s57` extension and only add the `.000` extension instead of including both?

Reply:
I changed it because the ENC files have a ".000" extension not .s57
From the spec, the .000 file is the base and then there is .001, .002 etc for updates

I'm not sure if we could include both file extensions.

Reply:
Uncommented that line ro re-add `*.s57` as an option